.github/workflows/propagate.yml

```diff
@@ -13,14 +13,7 @@ jobs:
   propagate:
     name: Propagate to affected version branches
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        go-version:
-          - 1.20.x
     env:
-      ACTIVESTATE_CI: true
```

Comment:
It makes sense to remove this one here but I think we should include it on workflows that use the state tool as consumers of our analytics could be using this now or may want to in the future.

Reply:
Just searched and it looks like it's all there. Sorry, when I saw it removed I didn't look in the other files.